src/test/java/no/unit/nva/model/ModelTest.java

```diff
@@ -307,12 +307,11 @@ private static PublicationContext generateReportContext()
             .build();
     }
 
-    private static PublicationContext generateJournalContext() throws InvalidIssnException, MalformedURLException {
-        return new Journal.Builder()
-            .withOnlineIssn("1111-1119")
-            .withPrintIssn("2222-2227")
-            .withTitle("The journal of mechanically separated meats")
-            .build();
+    private static PublicationContext generateUnconfirmedSeries() throws InvalidIssnException {
```

Comment:
Change the method name to be in sync with terminology used in the rest of the model.

Reply:
Removed in next PR